README.md

```diff
@@ -68,17 +73,19 @@ words.each do |word|
   end
 end
 ```
-
-Snippet 6 - Big O:
+# This is a O(n) because the each method has to go through each element to print them.
+that
+Snippet 6 - Big O: O(n)
 ```ruby
 numbers = [1,2,3,4,5,6,7,8,9,10]
 
 def print_array(array)
     array.each {|num| puts num}
 end
+Big O: O(n)
 ```
-
-Snippet 7 - Big O:
+# this is a O(log n) because the while loop is cutting the elements.
```

Comment:
This is actually O(n^2) because in the _worst_ case we have to visit every element for every element in the array. (both loops are O(n) )